app/assets/stylesheets/pageflow/editor/base.css.scss

```diff
@@ -122,6 +122,7 @@ main.ui-layout-pane {
   padding: 0;
   overflow: hidden;
   border: 0;
+  background-color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.